easybuild/easyconfigs/c/code-server/code-server-3.7.3.eb

```diff
@@ -0,0 +1,25 @@
+name = 'code-server'
+version = '3.7.3'
+
+homepage = 'https://github.com/cdr/code-server'
+description = """Run VS Code on any machine anywhere and access it in the browser."""
+
+toolchain = SYSTEM
+
+source_urls = ['https://github.com/cdr/code-server/releases/download/v%(version)s/']
+sources = ['code-server-%(version)s-linux-%(mapped_arch)s.tar.gz']
+checksums = [
+    {
+        'code-server-%(version)s-linux-amd64.tar.gz': '7a90f3171c9bc6f65266066e35cc34d48a032910c136ea21116d28f3d7214547', 
+        'code-server-%(version)s-linux-arm64.tar.gz': 'ed0e0aa296bc52bdd927cb388f321d5bd03f625316df02e0004652f0bb035dc0',
+    }
+]
+
+install_cmd = "cp -a code-server-%(version)s-linux-%(mapped_arch)s/* %(installdir)s"
+
+sanity_check_paths = {
```

Comment:
This and `install_cmd` can go into the easyblock too?

Also, we should run something like `code-server --help` as sanity check command?

Reply:
Done